Custom MinUI Theme Generator for muOS.py

```diff
@@ -277,9 +289,9 @@ def generatePilImageVertical(progress_bar,workingIndex,muOSSystemName,listItems,
         text_width = 2000*render_factor
         text = item[0][:]
         text_color = f"#{selected_font_hex}" if index == workingIndex else f"#{deselected_font_hex}"
-        if mergeBoxArt and boxArtDrawn and override_bubble_cut_var.get():
```

Comment:
mergeBoxArt is not needed here and was causing an issue on some games not being auto ellipsed.  This method already checks the overlay_box_art_var checkbox directly and boxArtDrawn to true if it merged box art so there is no need for it.

Reply:
Yeah, I put in mergeBoxArt first, then thought it was better to add in a check for each item rather than the whole folder, but i must have forgotten to take it out, Thanks for finding this!